pom.xml

```diff
@@ -21,8 +21,8 @@
     <scm>
         <url>https://github.com/vitalidze/chromecast-java-api-v2</url>
         <connection>scm:git:https://github.com/vitalidze/chromecast-java-api-v2.git</connection>
-      <tag>HEAD</tag>
-  </scm>
+        <tag>HEAD</tag>
```

Comment:
I am fine with you that a PR that adds OSGi compatibility should not change the indentation of some other lines.

Just one comment :wink: 

You are using an indentation step with of four spaces and the whole pom.xml is well formatted / indent except this two lines.
The tag "connection" and "tag" are sibling childs of scm and on the same level.
connection is using eight spaces, tag is using four spaces.
The open tag "scm" is using four spaces, the closing one two.

I have assumed that this is not done by purpose and want give you a consistent indentation of the whole XML.

I can revert this.